docs/pages/packaging.md

```diff
@@ -5,12 +5,29 @@ layout: default
+| Name                                                                  | Short description                                                                                                           | 🚦  |                     Used by                     |
+| --------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- | --- | :---------------------------------------------: |
+| [setuptools](https://setuptools.pypa.io)                              | A widely used build backend, used to configure a Python package.                                                            | 🟢  | [autodE](https://github.com/duartegroup/autodE) |
+| [setuptools-scm](https://github.com/pypa/setuptools_scm/)             | Provides automatic versioning Python packages. Also automatically adds all files under source control to the sdist / wheel. | 🟢  | [btrack](https://github.com/quantumjot/btrack)  |
+| [poetry](https://github.com/python-poetry/poetry)                     | A tool that lets you build and package your project as well as managing dependencies                                        | 🟠  |                                                 |
+| [twine](https://pypi.org/project/twine/)                              | Tool for publishing Python packages on PyPI.                                                                                | 🟠  |                                                 |
+| [bump2version](https://pypi.org/project/bump2version/)                | Tool for version-bumping your software. No longer maintained                                                                | 🔴  |                                                 |
+| [bump-my-version](https://github.com/callowayproject/bump-my-version) | Tool for version-bumping your software. Superseded by setuptools_scm                                                        | 🔴  |                                                 |
+
+# Building
+
+| Name                                                  | Short description                                                                                         | 🚦  |                         Used by                          |
+| ----------------------------------------------------- | --------------------------------------------------------------------------------------------------------- | --- | :------------------------------------------------------: |
+| [build](https://pypa-build.readthedocs.io/en/stable/) | Straightforward tool to build a Python package.                                                           | 🟢  |                                                          |
+| [cibuildwheel](https://cibuildwheel.readthedocs.io)   | Builds python wheels for the main operating systems on continuous integration runs (e.g. GitHub actions). | 🟠  | [streamtracer](https://github.com/dstansby/streamtracer) |
```

Comment:
What's the reason for cibuildwheen going from green to orange?

Reply:
Difficult to use according to @paddyroddy